ui/common/src/commonMain/kotlin/social/androiddev/common/utils/AsyncImage.kt

```diff
@@ -44,7 +45,7 @@ fun <T> AsyncImage(
             }.fold(
                 onSuccess = { it },
                 onFailure = { t ->
-                    t.printStackTrace()
+                    DodoLogger.w { "Error when loading image. e=${t.message}" }
```

Comment:
instead of the `stacktrace.message` I would put the whole stacktrace into the logger (as 1st param)

Reply:
doh - good catch, late night coding 🙈